OpenShift/helm/nexus-iq/values.yaml

```diff
@@ -35,44 +27,61 @@ fullnameOverride: ""
 serviceAccount:
   # Specifies whether a service account should be created
   create: true
+  # Annotations to add to the service account
+  annotations: {}
   # The name of the service account to use.
   # If not set and create is true, a name is generated using the fullname template
   name:
 
-deployment:
-  # Uncomment and modify this to run a command before starting the IQ Server
-  preStart:
-    command:    # '["/bin/rm", "-f", "/path/to/lockfile"]'
-  # Uncomment and modify this to run a command after starting the IQ Server
-  postStart:
-    command:    # '["/bin/sh", "-c", "ls"]'
-  terminationGracePeriodSeconds: 120
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
 
 ingress:
-  enabled: false
-  annotations: {}
+  enabled: true
+  annotations: {kubernetes.io/ingress.class: nginx}
     # kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
   path: /
-  hosts: []
-    # - nexus.local
+  hosts:
+      - host: iq-server.demo
+        paths: [/]
+      - host: admin.iq-server.demo
```

Comment:
oh good. I felt we were leaving the admin port unpublished previously. are these ok to be hard-coded?

Reply:
I think we were too, this is very close to what I do in Nginx in my docker env.